lib/sign.js

```diff
@@ -50,13 +52,13 @@ function sign (program, options, config) {
         };
         if (options.xpi) {
           logger.log("Signing XPI: " + options.xpi);
-          return _.assign(xpiConfig, {xpiPath: options.xpi});
+          return _.assign(xpiConfig, {xpiPath: options.xpi, removeUnsigned: false});
         } else {
           // Merge the program and sub-command options.
           var xpiOptions = _.assign({}, program, options);
           return config.createXPI(manifest, xpiOptions).then(function(xpiPath) {
```

Comment:
Actually, can you get `config.createXPI` to make the XPI in a temporary directory? I'm guessing it doesn't support that but if it does that'd be cool.